### PR TITLE
chore: add meta/open graph tags

### LINKED ISSUE #57: Support opengraph tags
#### Reason/Context

I shared the link on Twitter, and noticed we didn't have any open graph tags.

Be great to get some open graph tags for the conference for nice media sharing on things like Slack, Twitter etc.

### PR DESCRIPTION
**Pull request by mcturco**
**Description**

- Added general meta tags for title, description, type, url
- Added Open Graph tags for Facebook
- Added Twitter tags

**Related issue(s)**
Resolves #57

### CODE REVIEW

**Comment by derberg** [comment]
@mcturco that was fast! how was it with PUG 😄 

All is good except of the image. Look at Slack and Twitter:


Screenshot 2021-11-04 at 20 41 24
Screenshot 2021-11-04 at 20 41 48

docs say:
```
og:image The full URL for the image associated with the page—for example, https://yoursite.com/image.jpg rather than a relative URL like /image.jpg
```


try this in PUG config. URL is env variable that is populated by Netlify during the build, so it will work on production and previews too, no need to hardcode
```
image: `${process.env.URL}/images/meta/og-image.jpg`
```

**Comment by mcturco** [comment]
@derberg okay yup! I just noticed the same thing. It was just the image path that wasn't nearly correct once it passes through netlify :) you can see the preview here: https://www.opengraph.xyz/url/https%3A%2F%2Fdeploy-preview-71--peaceful-ramanujan-288045.netlify.app%2F/

**Comment by mcturco** [comment]
> how was it with PUG 😄

It was honestly not terrible! But probably would take some getting used to. I'm used to writing HTML the normal way, although with emmet abbreviations its not too bad 😄

File: sources/pug/_skeleton.pug [hunk 1]
```diff
@@ -11,9 +11,26 @@ html( lang='en', class= htmlClass )
...
+    // Primary Meta Tags
     meta( charset='utf-8' )
     meta( name='viewport', content='width=device-width, height=device-height, initial-scale=1.0' )
     meta( http-equiv='X-UA-Compatible', content='IE=edge' )
+    meta( name='title', content=ogData.title )
+    meta( name='description', content=ogData.description )
+
+    // Open Graph / Facebook
+    meta( property='og:type', content='website' )
+    meta( property='og:url', content='https://conference.asyncapi.com/' )
```
Comment (derberg) [inline sources/pug/_skeleton.pug:23]:
```suggestion
    meta( property='og:url', content=process.env.URL )
```

**Review by derberg — COMMENTED** [review]
oh, so the docs lie and you can actually have a relative path, interesting

anyway, just made one suggestion so we do not hardcode the URL of the conference website (as it will later turn into conference.2021.asyncapi.com and we forget to change)

**Comment by mcturco** [comment]
> oh, so the docs lie and you can actually have a relative path, interesting

I think it is due to Netlify deploying it. It must fill in the blanks for us 😄 

> just made one suggestion so we do not hardcode the URL

Good idea, thank you! Committed that in.

**Review by derberg — APPROVED** [review]
Perfect work 🚀  
Thank you so much 🍺

**Pull request merged**